definitions/npm/react-jss_v6.1.x/flow_v0.30.x-/react-jss_v6.1.x.js

```diff
@@ -8,6 +8,8 @@ declare module 'react-jss' {
 
   declare type Null = null | void;
 
+  declare type Dictionary = { [key: string]: {} }
```

Comment:
Looks like it is impossible in libdef to import type from another dependency - `jss`. At least I did not manage to make it work neither with `import type {JssStyle} from 'jss'` nor with `import type {JssStyle} from 'jss/lib/types'`